tools/tidb-binlog-monitor.md

```diff
@@ -0,0 +1,119 @@
+---
+title: TiDB Binlog 监控指标说明
+category: tools
+---
+
+# TiDB Binlog 监控指标说明
+
+目前对 TiDB Binlog 中一些比较重要的方面配置了监控，根据指标的重要程度分为了 Emergency、Critical 和 Warning 三种级别。
+
+## Emergency
+
+### 1. binlog_pump_loss_binlog_count
+
+监控规则
+
+changes(binlog_pump_loss_binlog_count[1m]) > 0
```

Comment:
this is confusing now after `EnableTolerant` is remove,  i think need to remove the metrics and use binlog_ pump_storage_error directly, and it don't mean data lost.